react/src/components/About.jsx

```diff
@@ -41,9 +41,7 @@ function About({ backend }) {
     // Error Handling
     response.forEach((r) => {
       if (!r.ok) {
-        Sentry.configureScope(function (scope) {
-          Sentry.setContext('response', r);
-        });
+        Sentry.getCurrentScope().setContext('response', r);
```

Comment:
let's instead make this:

```js
Sentry.withScope((scope) => {
  scope.setContext(...);
  Sentry.captureException(...);
});
```

This is the preferred pattern over using current scope like this!